tests/helpers/test_plates_helpers.py

```diff
@@ -361,19 +362,23 @@ def test_update_mlwh_with_cog_uk_ids(
 
         assert before_count == 5
 
+        test_timestamp = datetime.now().replace(second=0, microsecond=0)
         # run the function we're testing
         update_mlwh_with_cog_uk_ids(samples_for_mlwh_update)
 
         # check that the same samples in the MLWH now have the correct cog uk ids
         after = retrieve_samples_cursor(app.config, mlwh_sql_engine)
         after_count = 0
         after_cog_uk_ids = set()
+        after_update_timestamps = list()
         for row in after:
             after_count += 1
             after_cog_uk_ids.add(row[MLWH_LH_SAMPLE_COG_UK_ID])
+            after_update_timestamps.append(row[MLWH_LH_SAMPLE_UPDATED_AT].replace(second=0, microsecond=0))
 
         assert after_count == before_count
         assert after_cog_uk_ids == set(cog_uk_ids)
+        assert all([test_timestamp == updated_timestamp for updated_timestamp in after_update_timestamps])
```

Comment:
I think this seems a little dangerous, because if the test runs just towards the end of a second, and ticks over to the next second during the test, the equality test will fail - do you agree?

Trying to think of another approach... I guess you could check the timestamp before the update, and check it after, and just assert that it has changed?

Reply:
Yes I agree.  Testing that it has updated at all is good.  Patching `datetime.now()` to give you an expected timestamp is even better.